src/scene/vcModel.cpp

```diff
@@ -526,6 +540,13 @@ void vcModel::HandleContextMenu(vcState *pProgramState)
       {
         if (udFileExists(pProgramState->modelPath) != udR_Success || vcModals_OverwriteExistingFile(pProgramState->modelPath))
         {
+          if (!udStrchr(pProgramState->modelPath, "."))
+          {
+            // Adding a default externsion
+            size_t modelPathLength = udStrlen(pProgramState->modelPath);
+            udStrncpy(pProgramState->modelPath + modelPathLength, sizeof(pProgramState->modelPath), SupportedTileTypes_ExportDefault, udStrlen(SupportedTileTypes_ExportDefault));
+          }
```

Comment:
This can be greatly simplified:

```suggestion
          if (!udStrchr(pProgramState->modelPath, ".")) // Adding a default extension
            udStrcat(pProgramState->modelPath, SupportedTileTypes_ExportDefault);
```

Reply:
Fixed.